azure-pipelines.yml

```diff
@@ -139,8 +143,8 @@ jobs:
   - task: UseDotNet@2
     displayName: Install DotNet 2.1.x for signing tasks
     inputs:
-      packageType: 'sdk'
-      version: '2.1.x'
+      packageType: runtime
```

Comment:
`sdk` was more than required, so switching to `runtime` here presumably makes the pipeline run a little faster.